app/models/foreman_virt_who_configure/config.rb

```diff
@@ -247,6 +248,18 @@ def status_description
       _(STATUS_DESCRIPTIONS[status])
     end
 
+    def proxy_type
+      if proxy && proxy.url.match("^https")
+        "https_proxy"
+      else
+        "http_proxy"
+      end
+    end
+
+    def proxy
```

Comment:
Do we need this when there's already a belongs_to for http_proxy?

Reply:
I used this method because of default scope in HttpProxy.

Reply:
As there is no previous data, I can remove this method and will use http_proxy association.